pkg/network/ebpf/c/tracer.h

```diff
@@ -31,6 +31,7 @@ typedef struct {
     __u8 direction;
     __u64 sent_packets;
     __u64 recv_packets;
+    __u64 tags;
```

Comment:
Let's use a bitmask in this field for the "static" tags, so we can store up to 64 unique tags per connections.